pom.xml

```diff
@@ -1475,22 +1475,11 @@
                     <plugin>
                         <groupId>net.nicoulaj.maven.plugins</groupId>
                         <artifactId>checksum-maven-plugin</artifactId>
-                        <version>1.11</version>
                         <executions>
                             <execution>
                                 <id>source-release-checksum</id>
-                                <goals>
-                                    <goal>artifacts</goal>
-                                </goals>
                             </execution>
                         </executions>
-                        <configuration>
-                            <algorithms>
-                                <algorithm>SHA-512</algorithm>
-                                <algorithm>SHA3-512</algorithm>
-                            </algorithms>
-                            <csvSummary>false</csvSummary>
-                        </configuration>
                     </plugin>
```

Comment:
nit: can we remove this plugin def if it's already defined in the parent's `apache-release` profile?

Reply:
Done!